### PR TITLE
Universal Auth

### PR DESCRIPTION
**Pull request by robertu7**
* Add global `<UniversalAuthDialog>`;
* Add `views/UniversalAuth` for login and signup pages;
* Add `/me/settings/connect-wallet` page for logged-in user to bind wallet;
* Show "Connect Wallet" button on homepage if logged-in user didn't bind a wallet;
* Add wallet related error handling;

### CODE REVIEW

File: src/components/Forms/WalletAuthForm/Connect.tsx [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        }
+      >
+        <Form.List.Item title={maskAddress(values.address)} />
+
+        {errors.address && (
+          <Form.Field.Footer fieldMsgId={fieldMsgId} error={errors.address} />
+        )}
+      </Form.List>
+
+      <Form.CheckBox
```
Comment (tx0c) [inline src/components/Forms/WalletAuthForm/Connect.tsx:208]: if we want to differentiate the login/signup and not to show `tos` for login, may still need server side to add AuthResultType in generatingSigningMessage?

**Review by tx0c — APPROVED** [review]
avoid PR growing too big, good to merge now and see results on web-develop, may need another server change if wanted not-showing `tos` on login

**Review by devformatters — APPROVED** [review]
Go. 🧞

File: src/components/Forms/WalletAuthForm/Connect.tsx (hunk 1 above)
Comment (robertu7) [reply src/components/Forms/WalletAuthForm/Connect.tsx:208]: I think it's fine to leave the ToS checkbox in login. 🤔

cc @devformatters

**Pull request merged**